### PR TITLE
Adjust README to account for special cases like TLS and Docker secrets

### LINKED ISSUE #11: TLS support
Hi,

I create a TLS secret in a dedicated Namespace, with the type: kubernetes.io/tls

When I sync with Kubernetes-replicator, the secret is Opaque and is not working..

How to replicate with the correct type ?

### LINKED ISSUE #28: Not replicating .dockerconfigjson registry secrets
Simple and common use case: Replicate the registry credentials to all namespaces.

Typically this is a secret named "regcred" and it's data field is:

```
data:
  .dockerconfigjson: <base64 auth string here>
```

Seems that kubernetes-replicator does not copy these over.

### PR DESCRIPTION
**Pull request by martin-helmich**
Fixes #11
Fixes #28

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,20 @@
...
 
 This repository contains a custom Kubernetes controller that can be used to make
 secrets and config maps available in multiple namespaces.
 
+## Contents
+
+1. [Deployment](#deployment)
+    1. [Using Helm](#using-helm)
+    1. [Manual](#manual)
+1. [Usage](#usage)
```
Comment (Hermsi1337) [inline README.md]: `2.`

File: README.md [hunk 2]
```diff
@@ -1,11 +1,20 @@
...
-[![Docker Repository on Quay](https://quay.io/repository/mittwald/kubernetes-replicator/status "Docker Repository on Quay")](https://quay.io/repository/mittwald/kubernetes-replicator)
 [![Build Status](https://travis-ci.org/mittwald/kubernetes-replicator.svg?branch=master)](https://travis-ci.org/mittwald/kubernetes-replicator)
 
 This repository contains a custom Kubernetes controller that can be used to make
 secrets and config maps available in multiple namespaces.
 
+## Contents
+
+1. [Deployment](#deployment)
+    1. [Using Helm](#using-helm)
```
Comment (Hermsi1337) [inline README.md]: `2.`

File: README.md [hunk 3]
```diff
@@ -1,11 +1,20 @@
...
+    1. [Manual](#manual)
```
Comment (Hermsi1337) [inline README.md]: `3.`

File: README.md [hunk 4]
```diff
@@ -1,11 +1,20 @@
...
+1. [Usage](#usage)
+    1. [1. Create the source secret](#1-create-the-source-secret)
+    1. [2. Create empty secret](#2-create-empty-secret)
```
Comment (Hermsi1337) [inline README.md]: `2.`

File: README.md [hunk 5]
```diff
@@ -1,11 +1,20 @@
...
+    1. [Special case: TLS secrets](#special-case-tls-secrets)
```
Comment (Hermsi1337) [inline README.md]: `3.`

File: README.md (hunk 1 above)
Comment (martin-helmich) [reply README.md]: Not necessary. Markdown renders ordered lists automatically.

**Pull request merged**